Library/MagicalRecordStack/MagicalRecordStack.m

```diff
@@ -55,14 +55,17 @@ + (instancetype)stack
     return [[self alloc] init];
 }
 
-- (void)loadStack
-{
+- (void)loadStack{
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-variable"
     NSManagedObjectContext *context = [self context];
     NSString *stackType = NSStringFromClass([self class]);
     NSAssert(context, @"No NSManagedObjectContext for stack [%@]", stackType);
     NSAssert([self model], @"No NSManagedObjectModel loaded for stack [%@]", stackType);
     NSAssert([self store], @"No NSPersistentStore initialized for stack [%@]", stackType);
     NSAssert([self coordinator], @"No NSPersistentStoreCoodinator initialized for stack [%@]", stackType);
+#pragma GCC diagnostic pop
```

Comment:
I think this can potentially be moved back up underneath the `stackType` declaration. There's no need to disable it for the assertions — in general, I prefer to keep my clang diagnostic pragmas as minimal as possible, and only include _exactly_ what needs it and never the surrounding code.